lib/index.js

```diff
@@ -322,17 +325,28 @@ api.update = brCallbackify(async ({actor, id, patch, sequence}) => {
   const record = await api.get({actor, id});
 
   if(record.meta.sequence !== sequence) {
-    return new BedrockError(
+    throw new BedrockError(
       'Could not update Account. Record sequence does not match.',
       'InvalidStateError', {
         httpStatusCode: 409,
         public: true,
         actual: sequence,
         expected: record.meta.sequence
-      });
+      }
+    );
```

Comment:
lint rules shouldn't make us do this, original is what we want

Reply:
these are the options:

https://eslint.org/docs/rules/function-paren-newline

what is odd is that vue actually does allow the type of behavior you want with html elements inside template so it seems like someone out there would allow line-break then close on the same line as param.

Reply:
might be multiline

Reply:
Seems like we want "never" ... I don't think we have a case where we want to put a parenthesis on its own line by itself.

Reply:
I'll set it to never in this project and do a fix and see how it works.

Reply:
we actually do because with BedRockError some of those strings are so long that

```js
throw new BedrockError(
        'Account does not exist.', 'NotFoundError', {
          httpStatusCode: 404,
          public: true,
        }); // this errors with never and also with consistent

throw new BedrockError( 'Account does not exist and a really long string here.', 
  'NotFoundError', {
     httpStatusCode: 404,
      public: true,
    }); // lint error for line length being over 80
```

what we really need a func-paren-rule that allows
if a parameter's name + function declaration is over 80 then
allow a new line at the start of the declaration. do not allow a new line
at the end of the function paren.

another issue with the never rule is this

```js
  const result = await bd.database.collections.account.findOne( // eslint rule for line length
```